events/elastic-search.md

```diff
@@ -0,0 +1,17 @@
+# ElasticSearch, You Know For Search
+
+## What was the event about?
+
+Aravind Putrevu, a developer advocate for Elastic, gave a talk on the rising usage of ElasticSearch and also presented a demo
+on how easy it is to setup and run.
```

Comment:
Newline?

Reply:
Newline

Reply:
It is done as to provide confortable length of the line when editing it; too long lines are not good.

Reply:
I agree with Shivam. We'll mostly look at the [rendered version](https://github.com/kossiitkgp/docs/blob/3876356e0f2a2096c8ba7a2c2d56fc934caa1f95/events/elastic-search.md) instead of the code, and it is exactly how it should be.

We edit markdown files in our editors so, curtailing the column length makes sense to me.